contribute/index.markdown

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Since this is a markdown file, please use markdown syntax for headers and links.

Reply:
Hmm .... i was following the practice of existing pages, e.g. [this](https://raw.githubusercontent.com/void-linux/void-linux.github.io/master/packages/index.markdown). Is the source of such other pages incorrect? If so, i can not only use Markdown syntax here, but also submit a separate PR making the necessary changes to those other pages.

Reply:
The packages page is a little special since its driven by javascript.  I'd be fine for just this one page to change.

Reply:
Thanks, updated.